src/plugins/ui_stateful/filter_evaluation.ts

```diff
@@ -202,17 +201,12 @@ export class FilterEvaluationPlugin extends UIPlugin {
             });
           }
 
-          // In xlsx, filter header should ALWAYS be a string and should be unique in the table
-          const headerPosition = {
-            col: filter.col,
-            row: filter.rangeWithHeaders.zone.top,
-            sheetId,
-          };
-          const headerString = this.getters.getEvaluatedCell(headerPosition).formattedValue;
+          // In xlsx, column header should ALWAYS be a string and should be unique in the table
+          const headerString = this.getters.getEvaluatedCell(position).formattedValue;
```

Comment:
Why can we now use `position` directly instead of `headerPosition` ? Is it just a previous mistake, or is there a reason why we can now do it ?

Reply:
It was useless before, headerPoisiton is the same as position 🙃